### DISCUSSION TITLE
Automatically prompt to install recommended extensions

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#59] [Ideas]
We could automatically prompt users to install recommended extensions like Prettier, and ESLint.

I don't know if this is possible for other editors, but I'd be happy to create a PR for VS Code.

**Comment by sergiodxa** [comment]
You can use a [devcontainer](https://containers.dev/) to list extensions and when someone opens the app in a compatible editor it will prompt to install them.

**Reply by kentcdodds** [reply]
Thanks sergio! We just added this and it's sufficient: https://github.com/epicweb-dev/epic-stack/blob/main/.vscode/extensions.json

VSCode as an editor is honestly a part of the opinions of the stack, so I'm fine just focusing on that editor.
